pallets/msa/src/lib.rs

```diff
@@ -151,10 +151,11 @@ pub mod pallet {
 
 	/// Storage type for key to MSA information
 	/// - Key: AccountId
-	/// - Value: [`KeyInfo`](common_primitives::msa::KeyInfo)
+	/// - Value: [`MessageSourceId`]()
```

Comment:
This will autoconnect it in the docs if you leave off the `()` (that is only sometimes possible, and I'm still not sure all of when you can get away with it)
```suggestion
	/// - Value: [`MessageSourceId`]
```